igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 bol.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.config
+
+import groovy.transform.CompileStatic
+import org.hibernate.validator.constraints.NotEmpty
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+import javax.validation.constraints.NotNull
+
+/**
+ * Helper class to map masters in properties file into a validated property map
```

Comment:
I think this comment is a bit misleading, what is masters in this context?

Reply:
More nitpick: It's not really a map, it's just a standard type-safe Spring Boot config.

Reply:
As the GitLab implementation is pretty similar to the GitHub one we just copied this comment from https://github.com/spinnaker/igor/blob/master/igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitHubProperties.groovy#L27
Should we omit it?